dipy/boots/resampling.py

```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/python
 
-#import modules
+# import modules
 import time
-import sys, os, traceback, optparse
+import sys
+import os
+import traceback
+import optparse
```

Comment:
Yes - good call. Please do remove . Running `pyflakes` (https://pypi.python.org/pypi/pyflakes) or `flake8` ( https://pypi.python.org/pypi/flake8) on this should also tell you about unused modules, so you can verify that you are not removing something necessary.